### PR TITLE
Spell check and cleanup

### PR DESCRIPTION
**Pull request by sungam3r**
**What issue does this PR address?**

Just spell check and cleanup

**Does this PR introduce a breaking change?**

No

### CODE REVIEW

File: src/Serilog.Sinks.Console/Sinks/SystemConsole/Output/EventPropertyTokenRenderer.cs [hunk 1]
```diff
@@ -37,8 +37,7 @@ public EventPropertyTokenRenderer(ConsoleTheme theme, PropertyToken token, IForm
         public override void Render(LogEvent logEvent, TextWriter output)
         {
             // If a property is missing, don't render anything (message templates render the raw token here).
-            LogEventPropertyValue propertyValue;
-            if (!logEvent.Properties.TryGetValue(_token.PropertyName, out propertyValue))
+            if (!logEvent.Properties.TryGetValue(_token.PropertyName, out LogEventPropertyValue propertyValue))
```
Comment (nblumhardt) [inline src/Serilog.Sinks.Console/Sinks/SystemConsole/Output/EventPropertyTokenRenderer.cs]: `var`?

File: src/Serilog.Sinks.Console/Sinks/SystemConsole/Rendering/ThemedMessageTemplateRenderer.cs [hunk 2]
```diff
@@ -66,8 +66,7 @@ int RenderTextToken(TextToken tt, TextWriter output)
 
         int RenderPropertyToken(PropertyToken pt, IReadOnlyDictionary<string, LogEventPropertyValue> properties, TextWriter output)
         {
-            LogEventPropertyValue propertyValue;
-            if (!properties.TryGetValue(pt.PropertyName, out propertyValue))
+            if (!properties.TryGetValue(pt.PropertyName, out LogEventPropertyValue propertyValue))
```
Comment (nblumhardt) [inline src/Serilog.Sinks.Console/Sinks/SystemConsole/Rendering/ThemedMessageTemplateRenderer.cs]: `var`?

File: src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/SystemConsoleTheme.cs [hunk 3]
```diff
@@ -40,20 +40,18 @@ public class SystemConsoleTheme : ConsoleTheme
...
         public SystemConsoleTheme(IReadOnlyDictionary<ConsoleThemeStyle, SystemConsoleThemeStyle> styles)
         {
             if (styles == null) throw new ArgumentNullException(nameof(styles));
-            _styles = styles.ToDictionary(kv => kv.Key, kv => kv.Value);
+            Styles = styles.ToDictionary(kv => kv.Key, kv => kv.Value);
         }
 
         /// <inheritdoc/>
-        public IReadOnlyDictionary<ConsoleThemeStyle, SystemConsoleThemeStyle> Styles => _styles;
+        public IReadOnlyDictionary<ConsoleThemeStyle, SystemConsoleThemeStyle> Styles { get; private set; }
```
Comment (nblumhardt) [inline src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/SystemConsoleTheme.cs]: 👍 - if assignment only happens in the constructor, this doesn't need `private set` - ` { get; }` alone will do the job.

File: src/Serilog.Sinks.Console/Sinks/SystemConsole/Output/EventPropertyTokenRenderer.cs (hunk 1 above)
Comment (sungam3r) [reply src/Serilog.Sinks.Console/Sinks/SystemConsole/Output/EventPropertyTokenRenderer.cs]: Will not compile

Comment (sungam3r) [reply src/Serilog.Sinks.Console/Sinks/SystemConsole/Output/EventPropertyTokenRenderer.cs]: My mistake, it fits.

Comment (sungam3r) [reply src/Serilog.Sinks.Console/Sinks/SystemConsole/Output/EventPropertyTokenRenderer.cs]: fixed

File: src/Serilog.Sinks.Console/Sinks/SystemConsole/Rendering/ThemedMessageTemplateRenderer.cs (hunk 2 above)
Comment (sungam3r) [reply src/Serilog.Sinks.Console/Sinks/SystemConsole/Rendering/ThemedMessageTemplateRenderer.cs]: fixed

File: src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/SystemConsoleTheme.cs (hunk 3 above)
Comment (sungam3r) [reply src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/SystemConsoleTheme.cs]: fixed

**Comment by nblumhardt** [comment]
Thanks, looking good 👍 

(I personally consider `""` to be a nice terse shorthand for `string.Empty`, but happy enough either way 😄.)

**Comment by sungam3r** [comment]
I agree. Some StyleCop analyzer rules are controversial. String.Empty is just an obvious expression of intent.

**Pull request merged**